x-pack/plugins/security_solution/public/resolver/lib/aabb.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import * as vector2 from './vector2';
+import { AABB } from '../types';
+
+/**
+ * Return a boolean indicating if 2 vector objects are equal.
```

Comment:
:grey_question:  Some explanation of what `AABB` represents in the comments could be of help to a reader.

Reply:
So far i've tried to document the type: https://github.com/elastic/kibana/blob/700f53d3a331c9ec3307f76eaf08ea8caf38183b/x-pack/plugins/security_solution/public/resolver/types.ts#L162

This way, you can browse the types file and see what every concept we model is.
Also,  your editor can show you those comments on hover usually:
e.g. in mine: 
![image](https://user-images.githubusercontent.com/35559/85615862-e2500c80-b62a-11ea-98cd-5bcc35f2df5e.png)